doc/sphinx/user_manual/concepts/boundless_world.md

```diff
@@ -2,22 +2,22 @@
 The world is boundless
 ======================
 
-One of the things which often confuses new and potential users is that the GWB domain doesn't have any bounds and is meshless. To explain how this works, we will first talk about the Cartesian coordinate system and then talk about the spherical coordinate system.
+A concept that often confuses new and potential users is that the GWB domain doesn't have any bounds and is meshless. To explain how this works, we will first talk about the Cartesian coordinate system and then talk about the spherical coordinate system.
 
 ## Boundless Cartesian coordinate system
 
-Users of the world builder provide the location where they want to have some tectonic features. They do this through providing map view coordinates (x and y) through the input file. We will explain later how we get the z coordinate. This means that they can set their coordinates wherever they want in an infinite 2D plane. The reason this is not an issue for the world builder is that when the world builder computes what for example the temperature at a location is, it just checks if that feature is present at that location or not. So no need to limit the domain! 
+Users of the World Builder provide the locations where they want to have some tectonic features. They do this through providing map view coordinates (x and y) through the input file. We will explain later how we get the z coordinate. Coordinates can be set anywhere in an infinite 2D plane e.g., the origin (0,0) is not restricted to the "center" of the domain but all features share a common coordinate system. This is not an issue for the World Builder since when World Builder computes a property, e.g., temperature, at a location, it just checks whether a feature is present at that location. So no need to limit the domain! 
```

Comment:
```suggestion
Users of the World Builder provide the locations where they want to have some tectonic features. They do this through providing map view coordinates (x and y) through the input file. We will explain later how we get the z coordinate. Coordinates can be set anywhere in an infinite 2D plane. For example the origin (0,0) is not restricted to the "center" of the domain but all features share a common coordinate system. This is not an issue for the World Builder since when World Builder computes a property, e.g., temperature, at a location, it just checks whether a feature is present at that location. So there is no need to define or limit the size of the domain! 
```